lib/rbi/model.rb

```diff
@@ -82,6 +82,18 @@ def initialize(loc: nil)
     end
   end
 
+  # A comment representing a RBS signature prefixed with `#:`
+  class RBSComment < Comment
```

Comment:
The comment here says this class represents an RBS signature. However, the name is `RBSComment`. Given that we'll also have the concept of inline type assertion in comments, should we instead name this `RBSSignature`?

Or if this is intended to be used by any RBS comment, then we can change the comment on top of the class.

Reply:
Changed for `RBS type`